master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -978,6 +988,25 @@ private[celeborn] class Master(
     context.reply(UnregisterShuffleResponse(StatusCode.SUCCESS))
   }
 
+  def batchHandleUnregisterShuffles(
+      context: RpcCallContext,
+      applicationId: String,
+      shuffleIds: List[Integer],
+      requestId: String): Unit = {
+    val map = JavaUtils.newConcurrentHashMap[Integer, StatusCode]()
+    val shuffleKeys = new util.ArrayList[String]()
+    shuffleIds.foreach { shuffleId =>
+      val shuffleKey = Utils.makeShuffleKey(applicationId, shuffleId)
+      shuffleKeys.add(shuffleKey)
+    }
+    statusSystem.batchHandleUnRegisterShuffles(shuffleKeys, requestId)
+    logInfo(s"Unregister shuffle $shuffleKeys")
+    shuffleIds.foreach { shuffleId =>
+      map.put(shuffleId, StatusCode.SUCCESS)
+    }
+    context.reply(BatchUnregisterShuffleResponses(StatusCode.SUCCESS, map))
```

Comment:
```suggestion
    val shuffleKeys = shuffleIds.map(shuffleId => Utils.makeShuffleKey(applicationId, shuffleId)).asJava
    statusSystem.batchHandleUnRegisterShuffles(shuffleKeys, requestId)
    
    logInfo(s"Unregister shuffle $shuffleKeys")
    
    val map = shuffleIds.map(shuffleId => shuffleId -> StatusCode.SUCCESS).toMap.asJava
    context.reply(BatchUnregisterShuffleResponses(StatusCode.SUCCESS, map))
```
Another minor suggestion.
In a more concise Scala way for the implementation.

Reply:
Thanks!